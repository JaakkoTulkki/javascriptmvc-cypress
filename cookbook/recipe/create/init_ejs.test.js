var ejs = require('ejs');

test('it renders properly', () => {
  expect.assertions(1);
  const obj = {
    title: 'Cleaning Supplies',
    supplies: ['mop', 'broom', 'duster']
  };
  let actual = ''
  ejs.renderFile(__dirname +'/test.ejs', obj, undefined, (err, str) => {
    console.log(err, str);
    actual = str;
  });
  expect(actual).toMatchSnapshot();
});