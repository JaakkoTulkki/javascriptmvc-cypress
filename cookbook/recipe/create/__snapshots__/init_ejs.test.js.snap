// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders properly 1`] = `
"<ul>

   <li>mop</li>

   <li>broom</li>

   <li>duster</li>

</ul>"
`;
